From 11a083ca5c0bea5fd0dd0ec2992e8ec446917768 Mon Sep 17 00:00:00 2001
From: pdiopX <piotrx.diop@intel.com>
Date: Wed, 9 Jan 2013 11:53:55 +0100
Subject: Fix AudioPolicyManagerBase compilation warnings

BZ: 78670
[CLEAN]

This patch fixes -Wsign-compare -Wreorder and -Wenum-compare warnings during
AudioPolicyManagerBase.cpp compilation.

Category: aosp improvement
Domain: AudioComms-General
Origin: internal
Upstream-Candidate: yes
Change-Id: I9f80da5fda811c6a96263e67dc8ef2d03f508bd3
Orig-Change-Id: I0c49be0b4cc679bedfbddf2d087abfc11fb943b0
Signed-off-by: pdiopX <piotrx.diop@intel.com>
---
 audio/AudioPolicyManagerBase.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/audio/AudioPolicyManagerBase.cpp b/audio/AudioPolicyManagerBase.cpp
index d483188..53ee923 100644
--- a/audio/AudioPolicyManagerBase.cpp
+++ b/audio/AudioPolicyManagerBase.cpp
@@ -362,7 +362,7 @@ void AudioPolicyManagerBase::setPhoneState(int state)
         newDevice = hwOutputDesc->device();
     }
 
-    int delayMs = 0;
+    uint32_t delayMs = 0;
     if (isStateInCall(state)) {
         nsecs_t sysTime = systemTime();
         for (size_t i = 0; i < mOutputs.size(); i++) {
-- 
1.7.4.1

